packages/beacon-node/src/execution/engine/index.ts

```diff
@@ -17,7 +17,7 @@ export type ExecutionEngineOpts =
   | ({mode?: "http"} & ExecutionEngineHttpOpts)
```

Comment:
We could consider making `mode` required to ensure it is set in default options, else default values determined by `mode` will be empty

https://github.com/ChainSafe/lodestar/blob/568a956b97a8b0813f0c63259773ee1f7ee29616/packages/cli/src/options/beaconNodeOptions/execution.ts#L41

Reply:
Updated in latest commit (https://github.com/ChainSafe/lodestar/pull/4576/commits/8c01e0229dcf449633b35780d9191b17ce88b756), I think this makes more sense as `defaultOptions.executionEngine.mode === "http"` is anyhow only statically evaluated